src/index.ts

```diff
@@ -56,11 +56,10 @@ export class TachyonClient<OriginActor extends TachyonActor> {
         this.config = { ...defaultTachyonClientOptions, ...config };
 
         this.oauthClient = new OAuth2Client({
-            server: `http${this.config.ssl ? "s" : ""}://${this.getServerBaseUrl()}`, // TODO: https, discovery, allow specifying custom address,
-            clientId: config.clientId ?? "tachyon_client",
-            clientSecret: config.clientSecret,
-            authorizationEndpoint: "/authorize",
-            tokenEndpoint: "/token",
+            server: `http${this.config.ssl ? "s" : ""}://${this.getServerBaseUrl()}`,
+            clientId: "generic_lobby",
```

Comment:
Sorry missed before, I would keep allowing to override the default value of `client_id` and fix the comment above in line 26. Also config's `client_secret` is no longer used.

Reply:
Addressed in 5964f4